src/cbrkit/sim/collections.py

```diff
@@ -104,31 +108,61 @@ class dtw[V](SimPairFunc[Collection[V], float]):
             >>> sim([1, 2, 3], [3, 4, 5])
             0.14285714285714285
             >>> sim = dtw(distance_func=lambda a, b: abs(len(str(a)) - len(str(b))))
-            >>> sim(["a", "bb", "ccc", "ddd", "ee", "fff"], ["cffffffffffcj", "dfffffffffffffffffffffffffffffffffded"])
-            0.011235955056179775
+            >>> sim(["a", "bb", "ccc"], ["aa", "bbb", "c"])
+            0.25
+            >>> sim = dtw(distance_func=lambda a, b: abs(a - b))
+            >>> sim([1, 2, 3], [1, 2, 3, 4], return_alignment=True)
+            (0.5, [(1, 1), (2, 2), (3, 3), (3, 4)])
         """
 
-        distance_func: Callable[[V, V], float] | None = None
+        distance_func: Callable[[Any, Any], float] | None = None
 
         def __call__(
-            self,
-            x: Collection[Any] | np.ndarray,
-            y: Collection[Any] | np.ndarray,
-        ) -> float:
+                self,
+                x: Collection[Any] | np.ndarray,
+                y: Collection[Any] | np.ndarray,
+                return_alignment: bool = False
+        ) -> Union[float, Tuple[float, List[Tuple[Any, Any]]]]:
```

Comment:
This return type does not work in CBRkit. Instead, use a StructuredValue (for an example have a look at `SequenceSim` in this file). There you can encode additional information like a mapping.